File: web-front/src/main/java/com/isystk/sample/web/front/controller/api/v1/member/post/MemberPostDetailImageRestForm.java
package com.isystk.sample.web.front.controller.api.v1.member.post;

import java.io.Serializable;
import javax.validation.constraints.Digits;

public class MemberPostDetailImageRestForm implements Serializable {

  @Digits(integer = 9, fraction = 0)
  Integer imageId;

  public @Digits(integer = 9, fraction = 0) Integer getImageId() {
    return this.imageId;
  }

  public void setImageId(@Digits(integer = 9, fraction = 0) Integer imageId) {
    this.imageId = imageId;
  }
}

File: web-front/src/main/kotlin/com/isystk/sample/web/front/controller/api/v1/member/post/MemberPostDetailImageRestForm.kt
package com.isystk.sample.web.front.controller.api.v1.member.post

import lombok.Getter
import lombok.Setter
import java.io.Serializable
import javax.validation.constraints.Digits

@Setter
@Getter
class MemberPostDetailImageRestForm : Serializable {
    var imageId: @Digits(integer = 9, fraction = 0) Int? = null
}